src/functions-and-arrays.js

```diff
@@ -26,16 +48,69 @@ function sum() {}
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function averageNumbers(array) {
+  
+  if (array.length === 0) {
+    return null;
+  } else {
+    const average = sumNumbers(array) / array.length;
```

Comment:
I like that you re-use the function you already wrote, so far nobody else had this as part of their solution and I marked it in every single feedback. Good job :)